sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_client_async.py

```diff
@@ -42,14 +42,17 @@ class EventHubConsumerClient(ClientBaseAsync):
     The main goal of `EventHubConsumerClient` is to receive events from all partitions of an EventHub with
     load-balancing and checkpointing.
 
-    When multiple `EventHubConsumerClient` operate within one or more processes or machines targeting the same
-    checkpointing location, they will balance automatically.
+    When multiple `EventHubConsumerClient` of the same event hub and same consumer group operate within one or more
+    processes or machines targeting the same checkpointing location, they will balance the number of partitions each
+    `EventHubConsumerClient` processes.
```

Comment:
same comment in the `_consumer_client.py`

Reply:
Changed to Anna's comment